src/components/Button.scss

```diff
@@ -1,20 +1,76 @@
-// TODO Move to src/styles/colors.scss
-$text: #fff;
+@import '../styles/colors';
+@import '../styles/fonts';
+
+//TODO :Add colors to Colors.scss
```

Comment:
I was thinking of doing it in another pull request that will change color naming in general, in the whole app after that i start adding to colors.scss